package.json

```diff
@@ -2,26 +2,28 @@
   "name": "blocking-elements",
   "version": "0.0.2",
   "description": "A polyfill for the proposed blocking elments stack API",
-  "main": "dist/blocking-elements.js",
+  "main": "blocking-elements.js",
```

Comment:
should this be .ts?

Reply:
Oops, this shouldn't actually have changed.